src/style/_form.scss

```diff
@@ -107,8 +107,8 @@
       &.validation-error {
         margin-bottom: 0;
 
-        color: #9F3A38;
-        border-bottom: .125rem solid #E0B4B4;
+        color: #CF0001;
+        border-bottom: .0625rem solid #CF0001;
```

Comment:
Since you removed the extra pixel from `border-bottom`, you should also remove `margin-bottom: 0`